test-cases/tests/backup-restore/j07b-verify-cluster-sso-backup-and-restore.md

```diff
@@ -0,0 +1,71 @@
+---
+components:
+  - product-sso
+products:
+  - name: rhoam
+    environments:
+      - osd-post-upgrade
+      - osd-fresh-install
+    targets:
+      - 0.1.0
+      - 0.2.0
+      - 1.0.0
+estimate: 1h
+tags:
+  - destructive
+---
+
+# J07B - Verify Cluster SSO Backup and Restore
+
+## Description
+
+Note: this test should only be performed at a time it will not affect other ongoing testing, or on a separate cluster.
+
+## Steps
+
+### Postgres
+
+1. Login via `oc` as **kubeadmin**
+
+2. Verify Clients and Realms exist in postgres.
+
+Create a throwaway Postgres instance to access the RHSSO Postgres instance
+
+```sh
+cat << EOF | oc create -f - -n redhat-rhoam-operator
+  apiVersion: integreatly.org/v1alpha1
+  kind: Postgres
+  metadata:
+    name: throw-away-postgres
+    labels:
+      productName: productName
+  spec:
+    secretRef:
+      name: throw-away-postgres-sec
+    tier: development
+    type: workshop
+EOF
+```
+
```

Comment:
i think this is missing the step to into oc rsh <throw_away_postgres-pod-name> before the psql commands